src/EVM/Expr.hs

```diff
@@ -1224,7 +1224,6 @@ simplifyProp prop =
     go (PNeg (PNeg a)) = a
 
     -- solc specific stuff
-    go (PEq (IsZero (Eq a b)) (Lit 0)) = PEq a b
```

Comment:
Why was this removed?

Reply:
It was only moved lower, to the block about double negation, I think it fits there better?